modules/core/src/main/java/org/locationtech/jts/io/OrdinateFormat.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 Martin Davis.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.locationtech.jts.io;
+
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+
+/**
+ * Formats numeric values for ordinates
+ * in a consistent, accurate way.
+ * <p>
+ * The format has the following characteristics:
+ * <ul>
+ * <li>It is consistent in all locales (in particular, the decimal separator is always a period)
+ * <li>Scientific notation is avoided 
+ * <li>The maximum number of decimal places reflects the available precision
+ * </ul> 
+ * 
+ * @author mdavis
+ *
+ */
+public class OrdinateFormat
+{
+  /**
+   * The number of fraction digits to accommodate a reasonable ordinat value
+   */
+  public static final int MAX_FRACTION_DIGITS = 18;
+  
+  public static OrdinateFormat DEFAULT = new OrdinateFormat();
+
+  private DecimalFormat format;
+  
+  public static DecimalFormat createFormat(int maximumFractionDigits) {
+    // specify decimal separator explicitly to work in all locales
+    DecimalFormatSymbols symbols = new DecimalFormatSymbols();
+    symbols.setDecimalSeparator('.');
+    DecimalFormat format = new DecimalFormat("0", symbols);
+    format.setMaximumFractionDigits(maximumFractionDigits);
+    return format;
+  }
+
+  public OrdinateFormat() {
+    format = createFormat(MAX_FRACTION_DIGITS);
+  }
+
+  public OrdinateFormat(int maximumFractionDigits) {
+    format = createFormat(maximumFractionDigits);
```

Comment:
`format = createFormat(Math.min(MAX_FRACTION_DIGITS, maximumFractionDigits));`

Reply:
I guess could do this, but the current code leaves the option open to force more digits if desired.